WooCommerce/Classes/Authentication/Prologue/LoginProloguePages.swift

```diff
@@ -106,24 +106,32 @@ final class LoginProloguePageTypeViewController: UIViewController {
 
 private extension LoginProloguePageTypeViewController {
     func configureStackView() {
-        view.addSubview(stackView)
+        // Scroll view to contain all contents
+        let scrollView = UIScrollView()
+        view.addSubview(scrollView)
+        scrollView.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            scrollView.bottomAnchor.constraint(equalTo: view.bottomAnchor, constant: Constants.stackBottomMargin),
+            scrollView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            scrollView.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+            scrollView.topAnchor.constraint(equalTo: view.topAnchor),
+        ])
```

Comment:
nit: can probably be simplified using

```suggestion
        view.pinSubviewToAllEdges(scrollView, insets: .init(top: 0, left: 0, bottom: -Constants.stackBottomMargin, right: 0))
```

Reply:
Fixed in c88ff1b90051f62fc80bcc00d099295d501a4a1d.